scripts/wasmToB64.js

```diff
@@ -0,0 +1,7 @@
+// Converts WASM bytecode to JSON object to create JSON that can be loaded in ESM module without requiring `fetch`
+const wasmBytes = require('fs').readFileSync('../src.ts/wasm/rust_verkle_wasm_bg.wasm')
+const encoded = Buffer.from(wasmBytes, 'binary').toString('base64');
```

Comment:
Can we do an Uint8Array instead here? Since we've gotten rid of it everywhere else.

Reply:
This is just a build script and not packaged with the module.  We could do it as a Uint8Array but then we need to add additional dev dependencies to handle the bytes to string conversions that Nodejs `Buffer` gives us for free.

Are you good with leaving it as is?

Reply:
Ah then let's leave it as is